lib/ansible/modules/cloud/azure/azure_rm_loadbalancer.py

```diff
@@ -782,7 +765,35 @@ def exec_module(self, **kwargs):
                 load_balancing_rules=load_balancing_rules_param
             )
 
-            self.results['state'] = self.create_or_update_load_balancer(param)
+            if load_balancer:
+                new_dict = self.new_load_balancer.as_dict()
```

Comment:
according to az cli, load  balance could be updated: add properties, remove a properties. pls add it in update operation.
https://docs.microsoft.com/en-us/cli/azure/network/lb?view=azure-cli-latest#az-network-lb-update

Reply:
I think we should discuss exactly how it should work. right now in Ansible idempotency works as follows:
- get current state of the resource
- compare whether input parameters match this state
- update if supplied parameters are different than existing state
how should we change that process?